pymongoexplain/commands.py

```diff
@@ -116,12 +151,21 @@ def __init__(self, collection: Collection,
     def command_name(self):
         return "find"
 
+
 class FindAndModifyCommand(BaseCommand):
     def __init__(self, collection: Collection,
                  kwargs):
-        super().__init__(collection.name)
+        super().__init__(collection.name, kwargs.pop("collation", None))
         for key, value in kwargs.items():
-            self.command_document[key] = value
+            if key == "update" and kwargs.get("replacement", None) is not None:
```

Comment:
I think you can remove all the "replacement" references here because we never pass "replacement" through kwargs.

Reply:
Replacement is passed through kwargs for find_one_and_replace

Reply:
Nevermind, I was confused. Fixed it now.